builtin/mainmenu/dlg_config_world.lua

```diff
@@ -101,18 +101,24 @@ local function enable_mod(this, toset)
 		-- game mods can't be enabled or disabled
 	elseif not mod.is_modpack then
 		if toset == nil then
-			mod.enabled = not mod.enabled
-		else
-			mod.enabled = toset
+			toset = not mod.enabled
+		end
+		for i = 1, #modmgr.mods_by_name[mod.name] do
```

Comment:
Shorter:
```Lua
for i, mod_to_set in pairs(modmgr.mods_by_name[mod.name]) do
	mod_to_set.enabled = mod_to_set.modpack == mod.modpack and toset
end
```

Reply:
`ipairs` in this case. I'm aware, but I avoid `ipairs`, and for my own code, I also avoid `pairs`. The reasons are performance and garbage generation. Both `ipairs` and `pairs` create a closure that needs to be garbage-collected, and the function call overhead slows down the loop.

I know this is not performance-critical, I'm just used to these idioms. I'll send the amendment.